vector/src/main/java/im/vector/app/features/devtools/RoomDevToolActivity.kt

```diff
@@ -174,21 +176,22 @@ class RoomDevToolActivity : SimpleFragmentActivity(), FragmentManager.OnBackStac
         super.onDestroy()
     }
 
-    override fun onPrepareOptionsMenu(menu: Menu?): Boolean = withState(viewModel) { state ->
-        menu?.forEach {
-            val isVisible = when (it.itemId) {
-                R.id.menuItemEdit -> {
-                    state.displayMode is RoomDevToolViewState.Mode.StateEventDetail
-                }
-                R.id.menuItemSend -> {
-                    state.displayMode is RoomDevToolViewState.Mode.EditEventContent ||
-                            state.displayMode is RoomDevToolViewState.Mode.SendEventForm
+    override fun handlePrepareMenu(menu: Menu) {
+        withState(viewModel) { state ->
+            menu.forEach {
+                val isVisible = when (it.itemId) {
+                    R.id.menuItemEdit -> {
+                        state.displayMode is RoomDevToolViewState.Mode.StateEventDetail
+                    }
+                    R.id.menuItemSend -> {
+                        state.displayMode is RoomDevToolViewState.Mode.EditEventContent ||
+                                state.displayMode is RoomDevToolViewState.Mode.SendEventForm
+                    }
+                    else -> true
```

Comment:
This is quite dirty and would deserve a rework...

Reply:
[293a177](https://github.com/vector-im/element-android/pull/6436/commits/293a1771487d706111091d9d1d536439dcd5eea7)